alternator/executor.cc

```diff
@@ -443,6 +443,8 @@ static rjson::value generate_arn_for_index(const schema& schema, std::string_vie
 static rjson::value fill_table_description(schema_ptr schema, table_status tbl_status, service::storage_proxy const& proxy)
 {
```

Comment:
> tags that start with "aws:" are for internal use.

Leave aws: tags to aws.

Reply:
The benefit of following that convention is that users will know they will not use the same tags. The alternative is to develop our own convention of tags users cannot use.

Reply:
Don't we already have the convention of Alternator's private tags, those **not** copied from AWS, being called "alternator: something", not aws:something? What did we call the wrote isolation tag? The tag we use to store TTL? The tag to set tablets?

Reply:
system:ttl_attribute

Reply:
I don't have a strong opinion about the tag names, there's a convention in the API we could have used, I can make it a 'system:' something or 'alternator:' something

Reply:
I've renamed the labels in V3